src/config_legacy.cpp

```diff
@@ -670,6 +670,7 @@ bool ConfigUtils::fromLegacyStorage(Config& config)
         }
         SET_PROPERTY(gamepadOptions, invertXAxis, legacyGamepadOptions.invertXAxis);
         SET_PROPERTY(gamepadOptions, invertYAxis, legacyGamepadOptions.invertYAxis);
+        SET_PROPERTY(gamepadOptions, profileNumber, 1);
```

Comment:
You should not change `config_legacy.cpp`. The file only contains code to perform the config conversion from pre-Protobuf to Protobuf. Default initialization is already performed in `ConfigUtils::initUnsetPropertiesWithDefaults()`.

Reply:
You are correct, I will remove that.

Reply:
@mthiesen - I've corrected that and pushed a squashed rebase. I think this is ready for final review and merge.